--- dwdiff-2.1.2.orig/config.pkg
+++ dwdiff-2.1.2/config.pkg
@@ -125,11 +125,12 @@
 	fi
 
 	if [ "yes" = "${with_unicode}" ] ; then
+		: "${PKG_CONFIG:=pkg-config}"
 		{
       check_message "Checking for ICU pkg-config... "
 			{
-				echo "Running pkg-config --cflags --libs icu-uc" >> config.log
-				if pkg-config --cflags --libs icu-uc 2>&1 > /dev/null ; then
+				echo "Running $PKG_CONFIG --cflags --libs icu-uc" >> config.log
+				if $PKG_CONFIG --cflags --libs icu-uc 2>&1 > /dev/null ; then
 					check_message_result "yes"
 					true
 				else
@@ -156,15 +157,15 @@
 }
 EOF
 				clean .config.o
-				test_link "functions in libicu" ICUFLAGS="\`pkg-config --cflags icu-uc\`" ICULIBS="\`pkg-config --libs icu-uc\`"
+				test_link "functions in libicu" ICUFLAGS="\`$PKG_CONFIG --cflags icu-uc\`" ICULIBS="\`$PKG_CONFIG --libs icu-uc\`"
 			}
 		} || {
 			check_message_result "!! Could not compile with Unicode support. Try configuring with --without-unicode."
 			exit 1
 		}
 
-		ICUFLAGS="-DUSE_UNICODE \`pkg-config --cflags icu-uc\`"
-		ICULIBS="\`pkg-config --libs icu-uc\`"
+		ICUFLAGS="-DUSE_UNICODE \`$PKG_CONFIG --cflags icu-uc\`"
+		ICULIBS="\`$PKG_CONFIG --libs icu-uc\`"
 		if checkfunction "nl_langinfo" "nl_langinfo(CODESET);" "<langinfo.h>" ; then
 			ICUFLAGS="${ICUFLAGS} -DUSE_NL_LANGINFO"
 		fi
